planner/core/physical_plan_test.go

```diff
@@ -590,7 +531,7 @@ func (s *testPlanSuite) TestINMJHint(c *C) {
 		dom.Close()
 		store.Close()
 	}()
-	tk := testkit.NewTestKit(c, store)
+	tk := oldtestkit.NewTestKit(c, store)
```

Comment:
below this line, all the changes in this file is just renaming testkit to oldtestkit
oldtestkit pkg will be removed after migration finish